package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "appengine-cloudtasks",
+  "description": "Google App Engine Cloud Tasks example.",
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "private": true,
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "files": [
+    "*.js"
+  ],
+  "dependencies": {
+    "@google-cloud/tasks": "^3.0.0",
+    "body-parser": "^1.18.3",
+    "express": "^4.16.3",
```

Comment:
do we need express here? I don't think we are running any server, right?

Reply:
True, we don't need express. Already removed it.